From 032fc53d88eab343f2476fdda6541ce9059d145d Mon Sep 17 00:00:00 2001
From: Tomas Zeman <tzeman@volny.cz>
Date: Sun, 28 Oct 2007 23:30:16 +0100
Subject: [PATCH] bglibs-1.102-1
 * added

diff --git a/source/custom/bglibs/FrugalBuild b/source/custom/bglibs/FrugalBuild
new file mode 100644
index 0000000..4170a5f
--- /dev/null
+++ b/source/custom/bglibs/FrugalBuild
@@ -0,0 +1,23 @@
+# Maintainer: Tomas Zeman <tzeman@volny.cz>
+
+pkgname=bglibs
+pkgver=1.104
+pkgrel=1
+pkgdesc="One stop library package"
+url="http://untroubled.org/bglibs/"
+archs=(i686)
+up2date='lynx -dump -nolist $url|grep $pkgname|grep .tar.gz|head -1|sed -e "s/.*$pkgname-\(.*\)\.tar.gz.*$/\1/"'
+source=($url/$pkgname-$pkgver.tar.gz)
+sha1sums=('2286c5540e160ffbf79765c35f06c0651eecc777')
+
+build() {
+	Fcd $pkgname-$pkgver
+	echo "/usr/include/bglibs" > conf-include
+	echo "/usr/lib" > conf-lib
+	echo "/usr/bin" > conf-bin
+	echo "/usr/man" > conf-man
+
+	make || Fdie
+	make install_prefix=$Fdestdir install || Fdie
+}
+# vim: ft=sh
