From e24e28e510a1fcc561b7d21d270b56b64dc3932b Mon Sep 17 00:00:00 2001
From: Tomas Zeman <tzeman@volny.cz>
Date: Sun, 28 Oct 2007 23:30:41 +0100
Subject: [PATCH] cvm-0.82-1
 * added

diff --git a/source/custom/cvm/FrugalBuild b/source/custom/cvm/FrugalBuild
new file mode 100644
index 0000000..a9dee8b
--- /dev/null
+++ b/source/custom/cvm/FrugalBuild
@@ -0,0 +1,26 @@
+# Maintainer: Tomas Zeman <tzeman@volny.cz>
+
+pkgname=cvm
+pkgver=0.90
+pkgrel=1
+pkgdesc="Credential Validation Modules"
+depends=('bglibs>=1.100')
+url="http://untroubled.org/cvm/"
+archs=(i686)
+up2date='lynx -dump -nolist $url|grep $pkgname|grep .tar.gz|head -1|sed -e "s/.*$pkgname-\(.*\)\.tar.gz.*$/\1/"'
+source=($url/$pkgname-$pkgver.tar.gz)
+sha1sums=('659c0bc349460cf19442b0bcdc06fad439d57491')
+
+build() {
+	Fcd $pkgname-$pkgver
+	echo "/usr/include/bglibs" > conf-bgincs
+	echo "/usr/lib" > conf-bglibs
+	echo "/usr/bin" > conf-bin
+	echo "/usr/include" > conf-include
+	echo "/usr/lib" > conf-lib
+	echo "gcc -s -L/usr/lib" > conf-ld
+
+	make || Fdie
+	make install_prefix=$Fdestdir install || Fdie
+}
+# vim: ft=sh
