diet-runit-1.9.0

diff --git a/source/dietlibc/diet-runit/FrugalBuild b/source/dietlibc/diet-runit/FrugalBuild
new file mode 100644
index 0000000..c207822
--- /dev/null
+++ b/source/dietlibc/diet-runit/FrugalBuild
@@ -0,0 +1,51 @@
+# Maintainer: Tomas Zeman <tzeman@volny.cz>
+
+branch=diet
+pkgorig=runit
+pkgname=$branch-$pkgorig
+pkgver=1.9.0
+pkgrel=1
+pkgdesc="init replacement"
+url="http://smarden.org/runit/"
+archs=(i686)
+up2date='lynx -dump -nolist $url/install.html|grep $pkgorig|grep tar.gz|head -1|sed -e "s/.*$pkgorig-\(.*\)\.tar.gz.*$/\1/"'
+makedepends=(dietlibc)
+source=($url/$pkgorig-$pkgver.tar.gz)
+provides=(runit)
+sha1sums=('a5f6974ba2f5571e1d18eefb659140451840943c')
+
+build() {
+	mkdir -p $Fdestdir/package/admin
+	
+	mv $Fsrcdir/admin/$pkgorig-$pkgver $Fdestdir/package/admin
+	cd $Fdestdir/package/admin/$pkgorig-$pkgver
+	
+	echo "diet gcc $CFLAGS" > src/conf-cc
+	echo "diet gcc -s -static" > src/conf-ld
+	package/compile || return 1
+	
+	rm -rf compile
+	chown -R root.root $Fdestdir/package/
+
+	cd $Fdestdir/package/admin/ && ln -s $pkgorig-$pkgver $pkgorig
+	
+	mkdir -p $Fdestdir/usr/share/man/man8
+	cd $pkgorig && cp man/*.8 $Fdestdir/usr/share/man/man8
+	mkdir -p $Fdestdir/sbin
+	install -m750 $Fdestdir/package/admin/runit/command/runit-init $Fdestdir/sbin/runit-init
+	install -m750 $Fdestdir/package/admin/runit/command/runit $Fdestdir/sbin/runit
+
+	mkdir $Fdestdir/command && cd $Fdestdir/command
+	for i in `ls $Fdestdir/package/admin/$pkgorig/command`; do ln -sf /package/admin/$pkgorig/command/$i $i; done
+	
+	
+	mkdir -p $Fdestdir/etc/runit/runsvdir/default
+	mkdir $Fdestdir/var
+	ln -sf default $Fdestdir/etc/runit/runsvdir/current
+	ln -sf /etc/runit/runsvdir/default $Fdestdir/var/service
+
+	sed 's,/var/service,/etc/runit/runsvdir/default,' \
+		< $Fdestdir/package/admin/runit/etc/2 > $Fdestdir/sbin/runsvdir-start 
+	chmod 755 $Fdestdir/sbin/runsvdir-start
+}
+# vim: ft=sh
