qmail-autoresponder

diff --git a/source/dietlibc/diet-qmail-autoresponder/FrugalBuild b/source/dietlibc/diet-qmail-autoresponder/FrugalBuild
new file mode 100644
index 0000000..3042e9c
--- /dev/null
+++ b/source/dietlibc/diet-qmail-autoresponder/FrugalBuild
@@ -0,0 +1,35 @@
+# Maintainer: Tomas Zeman <tzeman@volny.cz>
+
+branch=diet
+pkgorig=qmail-autoresponder
+pkgname=$branch-$pkgorig
+pkgver=0.97
+pkgrel=1
+pkgdesc="simple program to automatically respond to emails"
+makedepends=(dietlibc diet-bglibs)
+url="http://untroubled.org/qmail-autoresponder/"
+archs=(i686)
+up2date='lynx -dump -nolist $url|grep $pkgorig|grep .tar.gz|head -1|sed -e "s/.*$pkgorig-\(.*\)\.tar.gz.*$/\1/"'
+source=($url/$pkgorig-$pkgver.tar.gz)
+provides=($pkgorig)
+sha1sums=('49d3a1f8fc960cb43229dff0e40f176b61a3b18b')
+
+build() {
+	DDIR=/usr/lib/diet
+	Fcd $pkgorig-$pkgver
+	echo "$Fdestdir/usr/bin" > conf-bin
+	echo "$Fdestdir/usr/man" > conf-man
+	echo "diet gcc $CFLAGS" > conf-cc
+	echo "diet gcc -s -static" > conf-ld
+	echo "$DDIR/include/bglibs" > conf-bgincs
+	echo "$DDIR/lib" > conf-bglibs
+
+	# don't build qmail-autoresponder-mysql
+	make qmail-autoresponder || Fdie
+
+	Fmkdir usr/bin
+	install -m755 qmail-autoresponder $Fdestdir/usr/bin || Fdie
+	Fmkdir usr/man/man1
+	install -m644 qmail-autoresponder.1 $Fdestdir/usr/man/man1 || Fdie
+}
+# vim: ft=sh
