diet-ucspi-tcp

diff --git a/source/dietlibc/diet-ucspi-tcp/FrugalBuild b/source/dietlibc/diet-ucspi-tcp/FrugalBuild
new file mode 100644
index 0000000..e368e1e
--- /dev/null
+++ b/source/dietlibc/diet-ucspi-tcp/FrugalBuild
@@ -0,0 +1,29 @@
+# Maintainer: Tomas Zeman <tzeman@volny.cz>
+
+branch=diet
+pkgorig=ucspi-tcp
+pkgname=$branch-$pkgorig
+pkgver=0.88
+pkgrel=1
+pkgdesc="command-line tools for building TCP client-server applications"
+makedepends=(dietlibc)
+url="http://cr.yp.to/ucspi-tcp.html"
+archs=(i686)
+up2date='lynx -dump -nolist http://cr.yp.to/$pkgorig/install.html|grep $pkgorig|grep tar.gz|head -1|sed -e "s/.*$pkgorig-\(.*\)\.tar.gz.*$/\1/"'
+source=(http://cr.yp.to/$pkgorig/$pkgorig-$pkgver.tar.gz)
+sha1sums=('793b4189795b563085602c030dd8aa0d206ddc0e')
+provides=($pkgorig)
+
+build() {
+	Fcd $pkgorig-$pkgver
+	echo "/usr" > conf-home
+	echo "diet gcc $CFLAGS" > conf-cc
+	echo "diet gcc -s -static" > conf-ld
+	make || return 1
+	for bin in tcpserver tcprules tcprulescheck argv0 recordio \
+		tcpclient *\@ tcpcat mconnect mconnect-io addcr \
+		delcr fixcrio rblsmtpd; do
+		install -m 755 -D $bin $Fdestdir/usr/bin/$bin
+	done
+}
+# vim: ft=sh
