diet-openssl

diff --git a/source/dietlibc/diet-openssl/FrugalBuild b/source/dietlibc/diet-openssl/FrugalBuild
new file mode 100644
index 0000000..b3ce39c
--- /dev/null
+++ b/source/dietlibc/diet-openssl/FrugalBuild
@@ -0,0 +1,44 @@
+# Maintainer: Tomas Zeman <tzeman@volny.cz>
+
+branch=diet
+pkgorig=openssl
+pkgname=$branch-$pkgorig
+pkgver=0.9.8h
+pkgrel=1
+pkgdesc="The Open Source toolkit for Secure Sockets Layer and Transport Layer Security; dietlibc version"
+makedepends=(dietlibc gcc sed)
+url="http://www.openssl.org"
+archs=(i686)
+up2date='lynx -dump -nolist $url/source|grep $pkgorig|grep .tar.gz|head -1|sed -e "s/.*$pkgorig-\(.*\)\.tar.gz.*$/\1/"'
+source=($url/source/$pkgorig-$pkgver.tar.gz)
+sha1sums=('ced4f2da24a202e01ea22bef30ebc8aee274de86')
+
+build() {
+	DDIR=/usr/lib/diet
+	Fcd $pkgorig-$pkgver
+	
+	cp Configure Configure.orig
+#	sed -e 's/gcc:-DL_ENDIAN -DTERMIO -O3 -fomit-frame-pointer -m486 -Wall/diet gcc:-DL_ENDIAN -DTERMIO -O3 -fomit-frame-pointer -nostdinc -march=i686 -Wall/' \
+#		-e 's/gcc:-DL_ENDIAN -DTERMIO -O3 -fomit-frame-pointer -mcpu=pentium -Wall/diet gcc:-DL_ENDIAN -DTERMIO -O3 -fomit-frame-pointer -nostdinc -march=i686 -Wall/' \
+#		-e 's/gcc:-DL_ENDIAN -DTERMIO -O3 -fomit-frame-pointer -mcpu=pentiumpro -Wall/diet gcc:-DL_ENDIAN  -DTERMIO -O3 -fomit-frame-pointer -nostdinc -march=i686 -Wall/' \
+#		Configure.orig > Configure
+	sed -e 's/gcc:-DL_ENDIAN -DTERMIO -O3 -fomit-frame-pointer -Wall/diet gcc:-DL_ENDIAN -DTERMIO -O3 -fomit-frame-pointer -Wall/' \
+	-e 's/-mcpu=pentium/-march=i686/' \
+	-e 's/-mcpu=pentiumpro/-march=i686/' \
+	Configure.orig > Configure
+  
+	# no-sse to compile test sha512t
+	./config --prefix=$DDIR --openssldir=/usr/lib/diet/etc/ssl \
+		no-shared no-dso no-sse2 no-sse2
+	
+	make all || Fdie
+	make test || Fdie
+	
+	make INSTALL_PREFIX=$Fdestdir install
+	
+	Frm $DDIR/etc
+	Fmkdir $DDIR/etc
+	ln -sf /etc/ssl $Fdestdir/$DDIR/etc/ssl
+	Frm $DDIR/lib/pkgconfig
+}
+# vim: ft=sh
