dmraid

diff --git a/source/custom/dmraid/FrugalBuild b/source/custom/dmraid/FrugalBuild
new file mode 100644
index 0000000..49bc6a7
--- /dev/null
+++ b/source/custom/dmraid/FrugalBuild
@@ -0,0 +1,30 @@
+# Taken from Archlinux
+
+pkgname=dmraid
+pkgver=1.0.0.rc14
+pkgrel=1
+pkgdesc="Device mapper RAID interface"
+url="http://people.redhat.com/~heinzm/sw/dmraid/"
+depends=(device-mapper)
+archs=('i686')
+groups=('custom')
+license=('GPL')
+up2date="lynx -dump $url/src| Flasttar"
+source=($url/src/$pkgname-$pkgver.tar.bz2)
+sha1sums=('2b3284db46a995967d88993ae5ae36b57c513bc4')
+
+build() {
+  cd $startdir/src/$pkgname/$pkgver
+
+  # create static bin
+  #./configure --enable-static_link --prefix=/usr
+  #make || return 1
+  #make DESTDIR=$startdir/pkg exec_prefix=$startdir/pkg/usr install
+  #mv $startdir/pkg/usr/sbin/dmraid $startdir/pkg/usr/sbin/dmraid.static
+
+  # create dynamic linked bin
+  ./configure --prefix=/usr
+  make || return 1
+  make DESTDIR=$startdir/pkg exec_prefix=$startdir/pkg/usr install
+  mkdir -p $startdir/pkg/var/lock/dmraid
+}
