drbd-module-0.7
diff --git a/source/custom/drbd-module/FrugalBuild b/source/custom/drbd-module/FrugalBuild
new file mode 100644
index 0000000..49eaf89
--- /dev/null
+++ b/source/custom/drbd-module/FrugalBuild
@@ -0,0 +1,36 @@
+# Maintainer: Tomas Zeman <tzeman@volny.cz>
+
+pkgorig=drbd
+pkgname=$pkgorig-module
+pkgver=0.7.25
+pkgrel=2
+pkgdesc="block device mirroring a whole block device via network; network raid-1 implementation"
+url="http://www.drbd.org"
+license="GPL"
+depends=(kernel-source gcc make "kernel=2.6.24")
+archs=(i686)
+install=drbd.install
+_F_scm_type="git"
+_F_scm_url="git://git.drbd.org/drbd-0.7.git"
+_F_scm_tag="fe0c05db16a4dbdc6cfcb20c68b2f61a7793df1a"
+Finclude scm
+
+#up2date="lynx -dump -nolist http://oss.linbit.com/drbd/|grep drbd-0.7|grep tar.gz|head -1|sed -e 's/^.*drbd-\(.*\)\.tar\.gz.*$/\1/'"
+#source=(http://oss.linbit.com/drbd/0.7/$pkgorig-$pkgver.tar.gz 'http://git.drbd.org/?p=drbd-0.7.git;a=commitdiff_plain;h=fe0c05db16a4dbdc6cfcb20c68b2f61a7793df1a')
+#sha1sums=('9f507ffd41306409357e1eb4e6f73e3671acad28')
+
+
+build() {
+	Funpack_scm
+	#Fcd $pkgorig-$pkgver
+	#mkdir -p $startdir/pkg/usr/src/$pkgname
+	#cp -aR drbd $startdir/pkg/usr/src/$pkgname
+	#cp -aR scripts $startdir/pkg/usr/src/$pkgname
+	Fcd || Fdie
+	cd drbd
+	make clean
+	make KDIR=/usr/src/linux-`uname -r` || Fdie
+	make install PREFIX=$startdir/pkg || Fdie
+}
+
+# vim: ft=sh
diff --git a/source/custom/drbd-module/drbd.install b/source/custom/drbd-module/drbd.install
new file mode 100644
index 0000000..06437f2
--- /dev/null
+++ b/source/custom/drbd-module/drbd.install
@@ -0,0 +1,41 @@
+
+# This is a default template for a post-install scriptlet.  You can
+# remove any functions you don't need (and this header).
+
+# arg 1:  the new package version
+pre_install() {
+  /bin/true
+}
+
+# arg 1:  the new package version
+post_install() {
+  post_upgrade $1
+}
+
+# arg 1:  the new package version
+# arg 2:  the old package version
+pre_upgrade() {
+  /bin/true
+}
+
+# arg 1:  the new package version
+# arg 2:  the old package version
+post_upgrade() {
+  DRBD_OBJ=/lib/modules/`uname -r`/kernel/drivers/block/drbd.ko
+  /sbin/depmod -a || /sbin/depmod -e $DRBD_OBJ
+}
+
+# arg 1:  the old package version
+pre_remove() {
+  /bin/true
+}
+
+# arg 1:  the old package version
+post_remove() {
+  /sbin/depmod -a
+}
+
+op=$1
+shift
+$op $*
+
