From 1fe8d9875967ec89f54382eec6a1a2b8a317eab9 Mon Sep 17 00:00:00 2001
From: Tomas Zeman <tzeman@volny.cz>
Date: Sun, 21 Oct 2007 21:43:36 +0200
Subject: [PATCH] drbd-tools-8.0.6-1-i686
 * added

diff --git a/source/custom/drbd-tools/FrugalBuild b/source/custom/drbd-tools/FrugalBuild
new file mode 100644
index 0000000..24ad938
--- /dev/null
+++ b/source/custom/drbd-tools/FrugalBuild
@@ -0,0 +1,26 @@
+# Maintainer: Tomas Zeman <tzeman@volny.cz>
+
+pkgorig=drbd
+pkgname=$pkgorig-tools
+pkgver=0.7.25
+pkgrel=1
+pkgdesc="block device mirroring a whole block device via network; network raid-1 implementation"
+url="http://www.drbd.org"
+license="GPL"
+archs=(i686)
+depends=(glibc)
+makedepends=(gcc bison flex m4)
+backup=(etc/drbd.conf)
+up2date="lynx -dump -nolist http://oss.linbit.com/drbd/|grep drbd-0.7|grep tar.gz|head -1|sed -e 's/^.*drbd-\(.*\)\.tar\.gz.*$/\1/'"
+source=(http://oss.linbit.com/drbd/0.7/$pkgorig-$pkgver.tar.gz)
+sha1sums=('9f507ffd41306409357e1eb4e6f73e3671acad28')
+
+
+build() {
+	Fcd $pkgorig-$pkgver
+	make clean tools || Fdie
+	make install-tools PREFIX=$startdir/pkg INITD=$startdir/pkg/etc/rc.d/ || Fdie
+	Fmv etc/rc.d/drbd etc/rc.d/rc.drbd
+}
+
+# vim: ft=sh
