guilt

diff --git a/source/custom/guilt/FrugalBuild b/source/custom/guilt/FrugalBuild
new file mode 100644
index 0000000..1778ba0
--- /dev/null
+++ b/source/custom/guilt/FrugalBuild
@@ -0,0 +1,20 @@
+# Maintainer: <tzeman@volny.cz>
+
+pkgname=guilt
+pkgver=0.31.2
+pkgrel=1
+pkgdesc="A Quilt-like interface on top of Git"
+url="http://www.kernel.org/pub/linux/kernel/people/jsipek/guilt/"
+depends=('git')
+makedepends=('asciidoc' 'xmlto')
+groups=('custom')
+archs=('i686' 'x86_64')
+up2date="lynx -dump $url|Flasttar"
+source=($url/$pkgname-$pkgver.tar.gz)
+sha1sums=('b19f0a32c80e895dbb938fff1cb4520291eb52b5')
+
+build() {
+	Fcd
+	Fmake
+	make install install-doc PREFIX=$Fdestdir/usr || Fdie
+}
