guilt-0.28

diff --git a/source/custom/guilt/FrugalBuild b/source/custom/guilt/FrugalBuild
new file mode 100644
index 0000000..478578f
--- /dev/null
+++ b/source/custom/guilt/FrugalBuild
@@ -0,0 +1,20 @@
+# Maintainer: <tzeman@volny.cz>
+
+pkgname=guilt
+pkgver=0.28
+pkgrel=1
+pkgdesc="A Quilt-like interface on top of Git"
+url="http://www.kernel.org/pub/linux/kernel/people/jsipek/guilt/"
+depends=('git')
+makedepends=('asciidoc')
+groups=('custom')
+archs=('i686' 'x86_64')
+up2date="lynx -dump $url|Flasttar"
+source=($url/$pkgname-$pkgver.tar.gz)
+sha1sums=('c8101fc990f64465d6428f4296801d221b1c6db8')
+
+build() {
+	Fcd
+	Fmake
+	make install install-doc PREFIX=$Fdestdir/usr || Fdie
+}
