ipsvd

diff --git a/source/dietlibc/ipsvd/FrugalBuild b/source/dietlibc/ipsvd/FrugalBuild
new file mode 100644
index 0000000..f5b382b
--- /dev/null
+++ b/source/dietlibc/ipsvd/FrugalBuild
@@ -0,0 +1,36 @@
+# Maintainer: Tomas Zeman <tzeman@volny.cz>
+
+pkgname=ipsvd
+pkgver=0.14.0
+pkgrel=1
+pkgdesc="internet protocol service daemons"
+url="http://smarden.org/$pkgname/"
+archs=(i686)
+up2date='lynx -dump -nolist $url/install.html|grep $pkgname|grep tar.gz|head -1|sed -e "s/.*$pkgname-\(.*\)\.tar.gz.*$/\1/"'
+makedepends=(dietlibc)
+source=($url/$pkgname-$pkgver.tar.gz)
+sha1sums=('b43fe1c8ed6d75c6e143ef7e667cf37b54ec3d47')
+
+build() {
+	mkdir -p $Fdestdir/package/net
+	
+	mv $Fsrcdir/net/$pkgname-$pkgver $Fdestdir/package/net
+	cd $Fdestdir/package/net/$pkgname-$pkgver
+	
+	echo "diet gcc $CFLAGS" > src/conf-cc
+	echo "diet gcc -s -static" > src/conf-ld
+	package/compile || Fdie
+	
+	rm -rf compile
+	chown -R root.root $Fdestdir/package/
+
+	cd $Fdestdir/package/net/ && ln -s $pkgname-$pkgver $pkgname || Fdie
+	
+	Fmkdir /usr/share/man/man8 /usr/share/man/man5 /usr/share/man/man7
+	cd $pkgname || Fdie
+	for i in 5 7 8; do cp man/*.$i $Fdestdir/usr/share/man/man$i; done || Fdie
+
+	Fmkdir /command && cd $Fdestdir/command
+	for i in `ls $Fdestdir/package/net/$pkgname/command`; do ln -sf /package/net/$pkgname/command/$i $i; done
+}
+# vim: ft=sh
