omail-admin

diff --git a/source/custom/omail-admin/.htaccess b/source/custom/omail-admin/.htaccess
new file mode 100644
index 0000000..978961b
--- /dev/null
+++ b/source/custom/omail-admin/.htaccess
@@ -0,0 +1,4 @@
+php_value session.bug_compat_42 1
+php_value session.bug_compat_warn 0
+php_value session.use_cookies 0
+php_value register_globals 1
diff --git a/source/custom/omail-admin/FrugalBuild b/source/custom/omail-admin/FrugalBuild
new file mode 100644
index 0000000..5f08d43
--- /dev/null
+++ b/source/custom/omail-admin/FrugalBuild
@@ -0,0 +1,29 @@
+# Maintainer: Tomas Zeman <tzeman@volny.cz>
+
+pkgname=omail-admin
+pkgver=1.2rc1
+pkgrel=1
+pkgdesc="PHP4 based qmail+vmailmgrd Web interface"
+_F_sourceforge_dirname="omail"
+Finclude sourceforge
+backup=(etc/$pkgname/config.php etc/$pkgname/vmailmgrquotas)
+url="http://omail.omnis.ch/omail-admin/"
+purl="http://sourceforge.net/project/showfiles.php?group_id=3658"
+depends=('php' 'libmysqlclient' 'qmail-autoresponder')
+archs=('i686')
+up2date="lynx -nolist -dump '$purl'|grep -i '$pkgname'|grep .tar.gz|head -1|sed 's/.*$pkgname-\(.*\).tar.gz .*/\1/'"
+sha1sums=('7f652ba56f6b4064efbbae57204f5d661fbe937a')
+
+build() {
+	www='/var/www'
+	Fmkdir $www/$pkgname /etc/$pkgname/vmailmgrd-tcp-hosts /usr/sbin /var/vmailstats
+	Fcd
+	find . -name 'CVS' |xargs rm -fr || Fdie
+	mv *.php *.inc images templates $Fdestdir$www/$pkgname || Fdie
+	cp scripts/README* . || Fdie
+	cp scripts/vmailstats.pl $Fdestdir/usr/sbin/vmailstats || Fdie
+	chmod 755 $Fdestdir/usr/sbin/vmailstats || Fdie
+	Fmv $www/$pkgname/config.php /etc/$pkgname/
+	Fsed '^$vmailmgrquota_file .*' "\$vmailmgrquota_file = '/etc/$pkgname/vmailmgrquotas';" $Fdestdir/etc/$pkgname/config.php
+	Fln /etc/$pkgname/config.php $www/$pkgname/config.php
+}
