perl-netaddr-ip

diff --git a/source/network-extra/spamassassin/FrugalBuild b/source/network-extra/spamassassin/FrugalBuild
index 268fe06..347f7b0 100644
--- a/source/network-extra/spamassassin/FrugalBuild
+++ b/source/network-extra/spamassassin/FrugalBuild
@@ -13,7 +13,7 @@ depends=('perl-html-parser>=3.56-2' 'perl-digest-sha1>=2.11-2' 'openssl' \
 	'perl-ip-country' 'perl-libwww' 'perl-io-socket-inet6>=2.52' \
 	'perl-mail-spf-query' 'perl-net-ident' 'perl-io-socket-ssl' \
 	'razor-agents>=2.84-2' 'perl-mail-dkim' 'perl-encode-detect>=1.00-2' 'perl>=5.10.0-2' \
-	'perl-net-ssleay>=1.32-2')
+	'perl-net-ssleay>=1.32-2' 'perl-netaddr-ip')
 groups=('network-extra')
 archs=('i686' 'x86_64')
 backup=(etc/conf.d/spamd etc/mail/spamassassin/local.cf)
