Upgrade for Frugalware 1.0

diff --git a/source/custom/guilt/FrugalBuild b/source/custom/guilt/FrugalBuild
index a51a965..b75ab72 100644
--- a/source/custom/guilt/FrugalBuild
+++ b/source/custom/guilt/FrugalBuild
@@ -1,7 +1,7 @@
 # Maintainer: <tzeman@volny.cz>
 
 pkgname=guilt
-pkgver=0.31.2
+pkgver=0.32.1
 pkgrel=2
 pkgdesc="A Quilt-like interface on top of Git"
 url="http://www.kernel.org/pub/linux/kernel/people/jsipek/guilt/"
@@ -11,7 +11,7 @@ groups=('custom')
 archs=('i686' 'x86_64')
 up2date="lynx -dump $url|Flasttar"
 source=($url/$pkgname-$pkgver.tar.gz)
-sha1sums=('b19f0a32c80e895dbb938fff1cb4520291eb52b5')
+sha1sums=('d5d6b995100b239ca49d47fc4d40eb5a5c39ef73')
 
 build() {
 	Fcd
diff --git a/source/network-extra/vpnc/FrugalBuild b/source/network-extra/vpnc/FrugalBuild
index 833169a..0f22f9d 100644
--- a/source/network-extra/vpnc/FrugalBuild
+++ b/source/network-extra/vpnc/FrugalBuild
@@ -3,7 +3,7 @@
 # Maintainer: Tomas Zeman <tzeman@volny.cz>
 
 pkgname=vpnc
-pkgver=0.5.1
+pkgver=0.5.3
 pkgrel=1
 pkgdesc="vpnc - client for cisco vpn concentrator"
 url="http://www.unix-ag.uni-kl.de/~massar/vpnc"
@@ -13,7 +13,7 @@ depends=('libgcrypt')
 backup=('etc/vpnc/default.conf')
 source=($url/$pkgname-$pkgver.tar.gz)
 up2date="lynx -dump $url|grep 'vpnc-.*tar.gz'|head -1|sed 's/.*-\(.*\).t.*/\1/'"
-sha1sums=('efc71dba03aa40945af35e4b074d99f922ff7ffd')
+sha1sums=('321527194e937371c83b5e7c38e46fca4f109304')
 
 build() {
 	Fmake PREFIX=/usr
