Upgrade for Frugalware 1.0

diff --git a/source/custom/guilt/FrugalBuild b/source/custom/guilt/FrugalBuild
index a51a965..b75ab72 100644
--- a/source/custom/guilt/FrugalBuild
+++ b/source/custom/guilt/FrugalBuild
@@ -1,7 +1,7 @@
 # Maintainer: <tzeman@volny.cz>
 
 pkgname=guilt
-pkgver=0.31.2
+pkgver=0.32.1
 pkgrel=2
 pkgdesc="A Quilt-like interface on top of Git"
 url="http://www.kernel.org/pub/linux/kernel/people/jsipek/guilt/"
@@ -11,7 +11,7 @@ groups=('custom')
 archs=('i686' 'x86_64')
 up2date="lynx -dump $url|Flasttar"
 source=($url/$pkgname-$pkgver.tar.gz)
-sha1sums=('b19f0a32c80e895dbb938fff1cb4520291eb52b5')
+sha1sums=('d5d6b995100b239ca49d47fc4d40eb5a5c39ef73')
 
 build() {
 	Fcd
diff --git a/source/custom/vmailmgr/FrugalBuild b/source/custom/vmailmgr/FrugalBuild
index 1bc31af..2cf3711 100644
--- a/source/custom/vmailmgr/FrugalBuild
+++ b/source/custom/vmailmgr/FrugalBuild
@@ -2,7 +2,7 @@
 
 pkgname=vmailmgr
 pkgver=0.96.9
-pkgrel=2
+pkgrel=3
 pkgdesc="Virtual e-mail domains/users management programs."
 url="http://www.vmailmgr.org"
 makedepends=('python' 'sed' 'linuxdoc-tools' 'texi2html')
diff --git a/source/dietlibc/diet-bglibs/FrugalBuild b/source/dietlibc/diet-bglibs/FrugalBuild
index 592f564..ffbfb08 100644
--- a/source/dietlibc/diet-bglibs/FrugalBuild
+++ b/source/dietlibc/diet-bglibs/FrugalBuild
@@ -3,21 +3,19 @@
 branch=diet
 pkgorig=bglibs
 pkgname=$branch-$pkgorig
-pkgver=1.104
-pkgrel=2
+pkgver=1.106
+pkgrel=1
 pkgdesc="One stop library package"
 makedepends=(dietlibc gcc)
 url="http://untroubled.org/bglibs/"
 archs=(i686)
 up2date='lynx -dump -nolist $url|grep $pkgorig|grep .tar.gz|head -1|sed -e "s/.*$pkgorig-\(.*\)\.tar.gz.*$/\1/"'
-source=($url/$pkgorig-$pkgver.tar.gz libtool-tag.patch)
-sha1sums=('2286c5540e160ffbf79765c35f06c0651eecc777' \
-          '63bbea48243e7e442e54701e90adcd5e89cb76bf')
+source=($url/$pkgorig-$pkgver.tar.gz)
+sha1sums=('5f1914299c0f91a3d5c83bf6c8c1d0b824ffe624')
 
 build() {
 	DDIR=/usr/lib/diet
 	Fcd $pkgorig-$pkgver
-	patch -p0 < $Fsrcdir/libtool-tag.patch || Fdie
 	echo "diet gcc $CFLAGS" > conf-cc
 	echo "diet gcc -s -static" > conf-ld
 	echo "$DDIR/include/bglibs" > conf-include
diff --git a/source/dietlibc/diet-bglibs/libtool-tag.patch b/source/dietlibc/diet-bglibs/libtool-tag.patch
deleted file mode 100644
index 2172023..0000000
--- a/source/dietlibc/diet-bglibs/libtool-tag.patch
+++ /dev/null
@@ -1,11 +0,0 @@
---- Makefile.orig	2006-10-30 22:11:21.000000000 +0100
-+++ Makefile	2007-10-25 00:00:46.000000000 +0200
-@@ -991,7 +991,7 @@
- 	  echo '#!/bin/sh'; \
- 	  echo 'source=$$1; shift'; \
- 	  echo 'base=`echo "$$source" | sed -e s:\\\\.c$$::`'; \
--	  echo exec libtool --mode=compile --silent `head -n 1 conf-cc` -I.  '-o $${base}.lo -c $$source $${1+"$$@"}'; \
-+	  echo exec libtool --mode=compile --tag=CC --silent `head -n 1 conf-cc` -I.  '-o $${base}.lo -c $$source $${1+"$$@"}'; \
- 	) >libcompile
- 	chmod 755 libcompile
- 
diff --git a/source/network-extra/vpnc/FrugalBuild b/source/network-extra/vpnc/FrugalBuild
index 833169a..0f22f9d 100644
--- a/source/network-extra/vpnc/FrugalBuild
+++ b/source/network-extra/vpnc/FrugalBuild
@@ -3,7 +3,7 @@
 # Maintainer: Tomas Zeman <tzeman@volny.cz>
 
 pkgname=vpnc
-pkgver=0.5.1
+pkgver=0.5.3
 pkgrel=1
 pkgdesc="vpnc - client for cisco vpn concentrator"
 url="http://www.unix-ag.uni-kl.de/~massar/vpnc"
@@ -13,7 +13,7 @@ depends=('libgcrypt')
 backup=('etc/vpnc/default.conf')
 source=($url/$pkgname-$pkgver.tar.gz)
 up2date="lynx -dump $url|grep 'vpnc-.*tar.gz'|head -1|sed 's/.*-\(.*\).t.*/\1/'"
-sha1sums=('efc71dba03aa40945af35e4b074d99f922ff7ffd')
+sha1sums=('321527194e937371c83b5e7c38e46fca4f109304')
 
 build() {
 	Fmake PREFIX=/usr
