vim: cscope

diff --git a/source/apps/vim/FrugalBuild b/source/apps/vim/FrugalBuild
index 011e484..9630e1f 100644
--- a/source/apps/vim/FrugalBuild
+++ b/source/apps/vim/FrugalBuild
@@ -19,7 +19,7 @@ up2date="lynx -dump $url/sources.php|grep bz2$|sed 's/.*-\(.*\)\.t.*/\1/;q'"
 source=(ftp://ftp.vim.org/pub/vim/unix/$pkgname-$pkgver.tar.bz2 \
 	gvim.xpm README.Frugalware \
         vim-vimrc.diff frugalware.diff)
-sha1sums=('76f6a5a97746db06bd4786984a116793a16e3e10' \
+sha1sums=('a4b6641ca528fada71ea77c998a441495ed4984c' \
           'f7262c7429c42b7c1ef9cf1eaa6aa12292933525' \
           '50e139d94557b2ca361fed2130af103c47a8d654' \
           '985911e5902d495897105cdd1570dfe47adfa254' \
@@ -40,7 +40,7 @@ build()
 	cd src
 	if Fuse $USE_GUI; then
 		# first build gvim
-		Fmake --with-x --enable-gui=gtk2 --enable-multibyte
+		Fmake --with-x --enable-gui=gtk2 --enable-multibyte --enable-cscope
 		make DESTDIR=$Fdestdir VIMRTDIR= installvimbin
 		Fmv /usr/bin/vim /usr/bin/gvim
 		Fln /usr/bin/gvim /usr/bin/gview
@@ -51,7 +51,7 @@ build()
 
 	# now vim
 	Fmake --localstatedir=/var/lib/$pkgname --disable-gpm --with-x=no \
-		--enable-gui=no --enable-multibyte
+		--enable-gui=no --enable-multibyte --enable-cscope
 	make DESTDIR=$Fdestdir VIMRTDIR= install || return 1
 	Fln vim /usr/bin/vi
 	Fcp /usr/share/vim/vimrc_example.vim /usr/share/vim/vimrc
