vim: cscope

diff --git a/source/apps/vim/FrugalBuild b/source/apps/vim/FrugalBuild
index 5a4a03e..ee5a166 100644
--- a/source/apps/vim/FrugalBuild
+++ b/source/apps/vim/FrugalBuild
@@ -39,7 +39,7 @@ build()
 	cd src
 	if Fuse $USE_GUI; then
 		# first build gvim
-		Fmake --prefix=/usr --with-x --enable-gui=gtk2 --enable-multibyte
+		Fmake --prefix=/usr --with-x --enable-gui=gtk2 --enable-multibyte --enable-cscope
 		make DESTDIR=$Fdestdir VIMRTDIR= installvimbin
 		Fmv /usr/bin/vim /usr/bin/gvim
 		Fln /usr/bin/gvim /usr/bin/gview
@@ -50,7 +50,7 @@ build()
 
 	# now vim
 	Fmake --localstatedir=/var/lib/$pkgname --disable-gpm --with-x=no \
-		--prefix=/usr --enable-gui=no --enable-multibyte
+		--prefix=/usr --enable-gui=no --enable-multibyte --enable-cscope
 	make DESTDIR=$Fdestdir VIMRTDIR= install || return 1
 	Fln vim /usr/bin/vi
 	Fcp /usr/share/vim/vimrc_example.vim /usr/share/vim/vimrc
